code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/aepui/style/SmallImageUIStyle.kt

```diff
@@ -11,31 +11,51 @@
 
 package com.adobe.marketing.mobile.aepcomposeui.aepui.style
 
-import androidx.compose.material.MaterialTheme
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.graphics.Color
+import androidx.compose.runtime.ReadOnlyComposable
 import androidx.compose.ui.text.TextStyle
-import com.adobe.marketing.mobile.aepuitemplates.SmallImageTemplate
+import androidx.compose.ui.unit.sp
+import com.adobe.marketing.mobile.aepcomposeui.AepUIConstants
 
 /**
  * Class representing the style for a small image AEP UI.
  *
- * @property titleTextStyle The text style for the title, if provided.
+ * @property titleTextStyle The custom text style for the title, if provided.
  */
+
 class SmallImageUIStyle(
-    private val titleTextStyle: TextStyle? = null
+    var titleAepTextStyle: AepTextStyle? = null,
+    var bodyAepTextStyle: AepTextStyle? = null,
+    var buttonAepButtonStyle: Array<AepButtonStyle?> = arrayOfNulls(3),
+    var buttonAepTextStyle: Array<AepTextStyle?> = arrayOfNulls(3),
```

Comment:
What do the list of 3 items signify here for `buttonAepButtonStyle` and `buttonAepButtonStyle` ?

Reply:
There can be a maximum of 3 buttons in the small image card. This list represents the style for each in the same order

Reply:
Can we make the mapping more specific and linked to the buttonId - with a `Map<String, AepButtonStyle>`, `Map<String, AepTextStyle>`